os_info/src/linux/lsb_release.rs

```diff
@@ -91,9 +92,10 @@ fn parse(output: &str) -> LsbRelease {
 
 #[cfg(test)]
 mod tests {
-    use super::*;
     use pretty_assertions::assert_eq;
 
+    use super::*;
```

Comment:
Any reasons to move this line? :upside_down_face:

Reply:
Nope, it's IDEA Rust plugin optimize import feature - didn't even notice it was moved :)